requirements-dev.txt

```diff
@@ -17,3 +17,5 @@ multidict==4.5.2
 chardet==3.0.4
 yarl==1.3.0
 aiomcache==0.6.0
+mypy==0.740
```

Comment:
Latest `mypy` version introduces some new errors that seem to come from implicit equality of `Session` (as a `MutableMapping`) to `dict`, for example in the tests there are a lot of `session == {}` assertions which with the latest `mypy` throw a `comparison-overlap` error (comparing `Session` to `Dict[x,x]`).

I'll look into fixing them (this may take a bit more :s ).

Reply:
Opted to `cast`ing the dict side of the comparisons to the appropriate `MutableMapping`, if you prefer ignoring them or centralizing the definition let me know :)